install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -57,20 +57,32 @@ spec:
           - zipkin:9411
         {{- if $.Values.global.proxy.envoyStatsd.enabled }}
           - --statsdUdpAddress
+        {{- if $.Values.global.mixerNamespace }}
```

Comment:
User can already customize the host/port - no need for another level of options.